analysis/wwz/get_wwz_yields.py

```diff
@@ -98,7 +98,121 @@
 
 
 
-
+JERC_LST = [
```

Comment:
We should either not have an explicit list for each year, or we should store this long list somewhere else. 

But this is minor, let's not address it right now.

Reply:
My initial thoughts on how to fix this is trying to get things symmetrized for each year, but this may be avoidable if we get the green light on total

Reply:
Yeah let's leave it for now, and revisit once we get a GL on Total (hopefully)